app/src/main/java/com/routinely/routinely/login/LoginScreen.kt

```diff
@@ -126,10 +134,8 @@ fun LoginScreen(
                             viewModel.loginWithEmailAndPassword(email = email, password = password)
```

Comment:
If you remove this viewModel, and pass this fun as parameter, you can use Preview again